roles/openshift_facts/library/openshift_facts.py

```diff
@@ -842,13 +860,11 @@ def get_openshift_version():
             version: the current openshift version
     """
     version = None
-
-    if os.path.isfile('/usr/bin/openshift'):
-        _, output, _ = module.run_command(['/usr/bin/openshift', 'version'])
+    o_path = search_path('openshift')
```

Comment:
Instead of searching the path, we could always just invoke the command and catch the exception if it isn't found. I'm not sure if module.run_command honors the environment though.

Also, we might run into a  situation where an atomic host has a modified search path, we may just want to check certain known locations instead.